apisix/admin/stream_routes.lua

```diff
@@ -142,8 +163,18 @@ function _M.delete(id)
         return 400, {error_msg = "missing stream route id"}
     end
 
+    local items,_ = routes()
     local key = "/stream_routes/" .. id
     -- core.log.info("key: ", key)
+    local refer_list = {}
```

Comment:
```suggestion
    local refer_list
```

we don't need to init `refer_list` here? it would be inited in `check_router_refer`.